tests/plt/plot_test.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
 
 import scipp as sc
-from scipp.plt import Plot, Figure, widgets, input_node, node
+from scipp.plt import Plot, Figure, widgets, input_node, node, plot
```

Comment:
I feel it is confusing to call this module `plt`. Can we find a clearer (temporary) name?

Reply:
I'm happy moving everything to an `experimental` subfolder. So put the code in `scipp.experimental.plotting` and name the function `scipp.experimental.plot`? Then we could add more things to the `experimental` submodule in the future, as we once discussed a while ago?

Reply:
@jl-wynen Do you have an opinion?

Reply:
Is the goal to eventually replace `scipp.plotting` and reuse the same name? If so, `scipp.experimental.plotting` sounds good.